### DISCUSSION TITLE
Welcome to fire-detection-from-images Discussions!

### DISCUSSION THREAD

**Discussion by robmarkcole** [robmarkcole/fire-detection-from-images#19] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nixizfan** [comment]
Excuse me, have any of you experts used this code to publish papers on flame recognition?
